### DISCUSSION TITLE
Some artwork for website

### DISCUSSION THREAD

**Discussion by MarkDowes** [inkstitch/inkstitch#2847] [General]
Hi there. I have a few artwork examples I created with inkstitch that could be added to the website if wanted, something a bit more artsy than the current examples,
https://www.instagram.com/p/C4Vxqulqk8c/
https://www.instagram.com/p/C291SBeChXQ/
https://www.instagram.com/p/C291udxiboX/

**Comment by kaalleen** [comment]
Wow, these are nice examples. Thank you for sharing and allowing us to use them on the website. It would be a nice addition to the artwork gallery... I'm not sure though how to transfer them from instagram to the website?

**Comment by MarkDowes** [comment]
No worries! Thanks to all that work hard on inkstitch, it has been fun getting to know the plugin. I hope it continues to grow, it is a great tool for artists, and the eye-watering cost embroidery software needs fighting.

Can I share the images here? lets find out.

![cyber-cat](https://github.com/inkstitch/inkstitch/assets/63742467/76d3d0f4-3c32-421c-bb9b-0fc9a912d19a)
![kaiba](https://github.com/inkstitch/inkstitch/assets/63742467/36c138f3-eeef-473a-894d-94391d4b1885)
![space-cat](https://github.com/inkstitch/inkstitch/assets/63742467/ae7b99ac-ae4b-4269-bfbb-c15352f98945)
![troll-1](https://github.com/inkstitch/inkstitch/assets/63742467/62da639c-1d50-4a88-b532-40c213d3a4c5)
![troll-2](https://github.com/inkstitch/inkstitch/assets/63742467/07d039a7-6f63-4b26-8107-e0dd6034099e)
![cow](https://github.com/inkstitch/inkstitch/assets/63742467/e420f4b5-649d-412a-9f3d-0bf5f263ac4b)

**Comment by kaalleen** [comment]
Nice, thanks. Here you go: https://inkstitch.org/tutorials/inspiration/#markdowes

Don't hesitate to ask if you want anything changed.

**Reply by MarkDowes** [reply]
Oh! Can you change the name to @the_alch please? Mark is a fake name I gave to my hotmail account in the early 2000's lol.

Otherwise looking great. Oh I suppose a link to my insta would be great, https://www.instagram.com/the_alch/

Thanks.
